saq/job.py

```diff
@@ -84,6 +89,10 @@ class Job:
     heartbeat: int = 0
     retries: int = 1
     ttl: int = 600
+    retry_delay: float = 0.0
+    retry_backoff: bool = False
```

Comment:
i think there are too many args

what do you think about combining retry_backoff and max

retry_backoff: float = 0.0 means no backoff
retry_backoff=10.0 means 10 second max, no need to support infinite retry

retry_jitter... is this really necessary?

Reply:
> retry_jitter... is this really necessary?

Backoff is kind of pointless without jitter: https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/

How about:

```
retry_backoff: Union[bool, float] = False
```

False means no backoff.
True means backoff with no max.
A number means backoff with a max.
And we always have jitter.

Reply:
perfect, i'd rather always have jitter

and we can do false, true, and float